File: week-5/week-5PriyanshiKanoujiya/SeriesShow.c
#include <stdio.h>  
#include <math.h>  
 
int main()  
{  
    int n,An,Bn;
    
    printf("To show sum of n no. of terms of a series S.\n");
    printf("Enter the number of terms: ");
    scanf("%d",&n);
    
    printf("S =");
    
    for(int i=1;i <= n;i++)
    {
        An = pow(-1,i+1)*(2+(i-1)*3);
        Bn = (7+(i-1)*4);
        printf("(%d/%d) +",An,Bn);
    }
    
}  
    

File: week-5/week5-VaishnaviRajput/seriesShow.java
import java.util.Scanner;
import java.lang.Math;

public class seriesShow {

    static void Series(int n){
        for(int i=0;i<n;i++){
            int sign= (int)Math.pow(-1,i);//to assign sign of series terms
            int numb=(2+3*i)*sign;//numerator of series terms
            int demo=7+4*i;//denominator of series terms
            if(sign!=-1 && i!=0){
                System.out.print("+");
            }
            System.out.print(numb+"/"+demo);
        }
      
    }
     public static void main(String[] args){
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter number of terms:");

        if(sc.hasNextInt()){
            int terms=sc.nextInt();
        System.out.print("S = ");
        Series(terms);
        }
        else{
            System.out.println("!!Invalid input given!!");
        }

    sc.close();
}
}
